cypress/e2e/tests/pages/explorer/more-resources/api/custom-resource-definitions.spec.ts

```diff
@@ -0,0 +1,170 @@
+import { CustomResourceDefinitionsPagePo } from '@/cypress/e2e/po/pages/explorer/custom-resource-definitions.po';
+import { generateCrdsDataSmall } from '@/cypress/e2e/blueprints/explorer/more-resources/api/custom-resource-definition-get';
+import * as jsyaml from 'js-yaml';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+
+const crdsPage = new CustomResourceDefinitionsPagePo('local');
+const crdName = `e2etests.${ +new Date() }.example.com`;
+const crdGroup = `${ +new Date() }.example.com`;
+
+describe('CustomResourceDefinitions', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, () => {
+  before(() => {
+    cy.login();
+  });
+
+  describe('List', { tags: ['@vai'] }, () => {
+    it('can create a crd and see it in list view', () => {
+      CustomResourceDefinitionsPagePo.navTo();
+      crdsPage.waitForPage();
+      crdsPage.create();
+
+      cy.readFile('cypress/e2e/blueprints/explorer/more-resources/api/custom-resource-definition.yml').then((crdYaml) => {
+      // convert yaml into json to update name and group values
+        const json: any = jsyaml.load(crdYaml);
+
+        json.metadata.name = crdName;
+        json.spec.group = crdGroup;
+
+        crdsPage.yamlEditor().set(jsyaml.dump(json));
+      });
+
+      cy.intercept('POST', 'v1/apiextensions.k8s.io.customresourcedefinitions').as('createCRD');
+      crdsPage.crdCreateEditPo().saveCreateForm().resourceYaml().saveOrCreate()
```

Comment:
Comment only. This is nice as it tests the create from yaml process, however it's probably best placed in a specific test of its own and instead we create the resource via API request (there's an endpoint that can handle yaml used by the `Import YAML` header button/modal)

Reply:
ah i was wondering this. thanks for calling it out. I'll dm you if i cant get it working